Doc/tools/extensions/availability.py

```diff
@@ -116,6 +119,14 @@ def parse_platforms(self) -> dict[str, str | bool]:
         return platforms
 
 
+def _print_platform(platform: str, version: str | bool) -> str:
+    if version is True:
+        return platform
+    if not version:
+        return sphinx_gettext("not {platform}").format(platform=platform)
+    return f"{platform} ≥ {version}"
```

Comment:
```suggestion
    return f"{platform} >= {version}"
```

How well supported is "≥"? Does it render well in pdf outputs?

Reply:
epub 
![image](https://github.com/user-attachments/assets/6b6a8ff3-787d-4b7d-a687-36af593d69e2)

Reply:
@AA-Turner could you please advise, is there a way to recognize epub build in the directive code and fallback to `>=`? Or should I rather drop the UTF-8 character from here?

Reply:
I find no mention of using "≥" or ">=" for versions in the Google, Microsoft and Apple style guides, and they say to use words such as "Use version 2.2 or later" and not "Use version 2.2 or higher":
 
* https://developers.google.com/style/word-list#later
* https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/later
* https://support.apple.com/en-gb/guide/applestyleguide/apsg51b3c806/web

Reply:
I agree with @hugovk . This again will require translation.

Reply:
I will add that `>=` is currently used.

![image](https://github.com/user-attachments/assets/644e0e0b-63c8-4493-adff-ce204b35e935)

Reply:
> * https://developers.google.com/style/word-list#later

I find Google's style guide contradictory 🤔 

> A release with the highest version number might not be the latest version. For example, if version 2.0 of an operating system receives a bug-fix update after version 3.0 has been released, then version 2.0.1 might be the latest version, even though its version number is lower than 3.0.

And later they recommend *higher* in Android docs. I've filed a feedback to this section.

I agree that the written form may be easier to read than the symbolic one.

> I will add that >= is currently used.

I wonder what should be the approach, if we decide to use "Linux 2.6.27 or later", should we rewrite also the availability directives content in .rsts? Or introduce a simplified "syntax", that will be rendered to "or later" from a ">=" specificator? That may be confusing to docs editors. (?)

Reply:
If we choose to use words, I'd rewrite the existing ">=" to avoid problems/confusion by automation changing things in unexpected ways.

Reply:
We discussed this in this month's docs workgroup meeting: let's keep the directives compact; they're not prose, and use the single symbol. The Unicode symbol was introduced in 1993, hopefully it renders well on near anything. If there's an issue with Unicode in some of the backends (such as man pages), we should fix that.

(Minutes will appear at https://docs-community.readthedocs.io/en/latest/monthly-meeting/index.html in the near future.)